content/learn/book/welcome/plugins/_index.md

```diff
@@ -1,16 +1,150 @@
+These might provide basic windowing functionality, handle input or sound, calculate physics or provide game-specific logic like a tile map system.
+
+By combining Bevy's first-party plugins with third-party alternatives, you can modularly customize the behaviour of the Bevy game engine, and then add your own plugins to contain your own game-specific code in a well-organized way.
 
 ## `MinimalPlugins`
 
+Unless you're doing something very unusual, you will *always* want to include the plugins provided by Bevy's [`MinimalPlugins`](https://docs.rs/bevy/latest/bevy/struct.MinimalPlugins.html). We can click through to the source for the `impl PluginGroup for MinimalPlugins` to see what this adds:
+
+```rust
+use bevy_app::{PluginGroup, PluginGroupBuilder};
+
+pub struct MinimalPlugins;
+
+impl PluginGroup for MinimalPlugins {
+    fn build(&mut self, group: &mut PluginGroupBuilder) {
+        group.add(bevy_core::CorePlugin::default());
+        group.add(bevy_app::ScheduleRunnerPlugin::default());
+    }
+}
+```
+
+The `CorePlugin` handles low-level fundamentals such as updating app time, while the `ScheduleRunnerPlugin` sets up the main game loop to run repeatedly over time.
+
 ## `DefaultPlugins`
 
+Bevy's `DefaultPlugins` are intended to get up and running with a "standard game" as quickly as possible. Let's take a look at the source again:
+
+```rust
+pub struct DefaultPlugins;
+
+impl PluginGroup for DefaultPlugins {
+    fn build(&mut self, group: &mut PluginGroupBuilder) {
+        group.add(bevy_log::LogPlugin::default());
+        group.add(bevy_core::CorePlugin::default());
+        group.add(bevy_transform::TransformPlugin::default());
+        group.add(bevy_diagnostic::DiagnosticsPlugin::default());
+        group.add(bevy_input::InputPlugin::default());
+        group.add(bevy_window::WindowPlugin::default());
+        group.add(bevy_asset::AssetPlugin::default());
+        group.add(bevy_scene::ScenePlugin::default());
+
+        #[cfg(feature = "bevy_render")]
+        group.add(bevy_render::RenderPlugin::default());
+
+        #[cfg(feature = "bevy_sprite")]
+        group.add(bevy_sprite::SpritePlugin::default());
+
+        #[cfg(feature = "bevy_pbr")]
+        group.add(bevy_pbr::PbrPlugin::default());
+
+        #[cfg(feature = "bevy_ui")]
+        group.add(bevy_ui::UiPlugin::default());
+
+        #[cfg(feature = "bevy_text")]
+        group.add(bevy_text::TextPlugin::default());
+
+        #[cfg(feature = "bevy_audio")]
+        group.add(bevy_audio::AudioPlugin::default());
+
+        #[cfg(feature = "bevy_gilrs")]
+        group.add(bevy_gilrs::GilrsPlugin::default());
+
+        #[cfg(feature = "bevy_gltf")]
+        group.add(bevy_gltf::GltfPlugin::default());
+
+        #[cfg(feature = "bevy_winit")]
+        group.add(bevy_winit::WinitPlugin::default());
+
+        #[cfg(feature = "bevy_wgpu")]
+        group.add(bevy_wgpu::WgpuPlugin::default());
+    }
+}
+```
```

Comment:
I feel like this snippet is a bit too long to read. Since not every single detail is necessary here, maybe removing/hiding some lines could be a solution (I don't know if prepending a # to a line hides it like in rustdoc though).

Reply:
Yeah, I'm not sure I love linking this code directly.

Reply:
Yeah, that would be awesome. It would also remove the code/docs synchronization problem.